chainer/dataset/convert.py

```diff
@@ -3,6 +3,12 @@
 
 from chainer import cuda
 
+try:
+    import cupy
```

Comment:
Use `cuda.cupy` in the code instead of "try-import"ing cupy.

Reply:
All right, will stop using "try-import" there.